src/artbox/cli.py

```diff
@@ -1,135 +1,254 @@
+    """Process commands for specific flags; otherwise, show the help menu."""
+    if version:
+        typer.echo(f"Version: {__version__}")
+        raise typer.Exit()
+
+    if ctx.invoked_subcommand is None:
+        typer.echo(ctx.get_help())
+        raise typer.Exit(0)
+
+
+@app_voice.command("text-to-speech")
+def text_to_speech(
+    title: Annotated[
+        str, typer.Option("--title", help="Specify the name of the audio file")
+    ] = "artbox",
+    text_path: Annotated[
+        str,
+        typer.Option("--text-path", help="Specify the path of the text file"),
+    ] = "",
+    output_path: Annotated[
+        str,
+        typer.Option(
+            "--output-path", help="Specify the path to store the audio file"
         ),
-    )
-    parser.add_argument(
-        "method",
-        default=None,
-        help=("Specify the runner method to be performed."),
-    )
-    return parser
-
-
-def extract_artbox_args():
-    """Extract artbox arguments from the CLI call."""
-    artbox_args = {}
-    index_to_remove = []
-
-    for ind, arg in enumerate(list(sys.argv)):
-        if arg in [
-            "--help",
-            "--version",
-        ]:
-            continue
-
-        if not arg.startswith("--"):
-            continue
-
-        index_to_remove.append(ind)
-
-        arg_name = None
-        arg_value = None
-
-        next_ind = ind + 1
-
-        arg_name = sys.argv[ind][2:]
-
-        if (
-            len(sys.argv) == next_ind
-            or len(sys.argv) > next_ind
-            and sys.argv[next_ind].startswith("--")
-        ):
-            arg_value = True
-        else:
-            arg_value = sys.argv[next_ind]
-            index_to_remove.append(next_ind)
-
-        artbox_args[arg_name] = arg_value
-
-    # remove exclusive artbox flags from original sys.argv
-    for ind in sorted(index_to_remove, reverse=True):
-        sys.argv.pop(ind)
-
-    return artbox_args
-
-
-def show_version():
-    """Show version."""
-    print(__version__)
-
-
-def app():
-    """Call the artbox program with the arguments defined by the user."""
-    artbox_args = extract_artbox_args()
-
-    args_parser = _get_args()
-    args = args_parser.parse_args()
-
-    if args.version:
-        return show_version()
-
-    if args.runner == "sound":
-        runner = Sound(artbox_args)
-    elif args.runner == "voice":
-        runner = Voice(artbox_args)
-    elif args.runner == "video":
-        runner = Video(artbox_args)
-    elif args.runner == "youtube":
-        runner = Youtube(artbox_args)
-
-    return getattr(runner, args.method.replace("-", "_"))()
+    ] = "",
+    engine: Annotated[
+        str,
+        typer.Option(
+            "--engine",
+            help="Choose the text-to-speech engine (Options: edge-tts, gtts)",
+        ),
+    ] = "edge-tts",
+    lang: Annotated[
+        str,
+        typer.Option(
+            "--lang", help="Choose the language for audio generation"
+        ),
+    ] = "",
```

Comment:
```suggestion
    ] = "en",
```

it is also necessary to define its default as "en"

Reply:
Done.